src/log.js

```diff
@@ -382,15 +383,7 @@ export class Log {
    */
   prepareError_(error) {
     error = duplicateErrorIfNecessary(error);
-    if (this.suffix_) {
-      if (!error.message) {
-        error.message = this.suffix_;
-      } else if (error.message.indexOf(this.suffix_) == -1) {
-        error.message += this.suffix_;
-      }
-    } else if (isUserErrorMessage(error.message)) {
-      error.message = error.message.replace(USER_ERROR_SENTINEL, '');
-    }
+    error.userError = this.userMode.userError || false;
```

Comment:
You already did `this.userMode.userError = opt_userError || false`, no need to add the `|| false` here